src/Game.js

```diff
@@ -160,7 +160,15 @@ class Game extends Component {
         }
 
         const className = classList.join(" ");
-        rowInnerMarkup.push(<td key={`${i}-${j}`} className={className} />);
+        let test;
```

Comment:
Instead of checking existence of classes using indexOf why not use the already conditionals which are already present like these on line 154 - 
```js
        if (this.spritesPos[i] === j) {
          classList.push("has-sprite");
        }

        if (userPosY === i && userPosX === j) {
          classList.push("has-user");
        }
```